.npmrc

```
@@ -0,0 +1,8 @@
+# If set to false, then ignore package-lock.json files when installing. This will also prevent writing package-lock.json if save is true. source: https://docs.npmjs.com/cli/v11/using-npm/config
+package-lock=true
+# If true, npm does not run scripts specified in package.json files. source: https://docs.npmjs.com/cli/v11/using-npm/config#ignore-scripts
+ignore-scripts=true
```

```suggestion
ignore-scripts=true
```

This would break essential tools, such as `husky`. We need to integrate a solution that allows us to provide whitelists.